.github/workflows/pack.yml

```
@@ -0,0 +1,99 @@
+name: Pack
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+  release:
+    types:
+      - released
+
+env:
+  NODE_PREBUILD_CMD: npx prebuild -t 10.0.0 -t 12.0.0 -t 14.0.0 -t 16.0.0 -t 18.0.0 -t 20.0.0 --strip -u ${{ secrets.GH_TOKEN }}
+  ELECTRON_PREBUILD_CMD: npx prebuild -r electron -t 3.0.0 -t 4.0.0 -t 5.0.0 -t 6.0.0 -t 7.0.0 -t 8.0.0 -t 9.0.0 -t 10.0.0 -t 11.0.0 -t 12.0.0 -t 13.0.0 -t 14.0.0 -t 15.0.0 -t 16.0.0 -t 17.0.0 -t 18.0.0 -t 19.0.0 -t 20.0.0 -t 21.0.0 -t 22.0.0 -t 23.0.0 -t 24.0.0 -t 25.0.0 --strip -u ${{ secrets.GH_TOKEN }}
```

The `openssl_fips != ""` error 

https://github.com/tree-sitter/tree-sitter-bash/actions/runs/5948296886/job/16131941716

happens when prebuilding for an old Node version. You can either not prebuild for those versions and just do the latest supported versions

https://github.com/nodejs/Release

https://www.electronjs.org/docs/latest/tutorial/electron-timelines

```suggestion
  NODE_PREBUILD_CMD: npx prebuild -t 16.0.0 -t 18.0.0 -t 20.0.0 --strip -u ${{ secrets.GH_TOKEN }}
  ELECTRON_PREBUILD_CMD: npx prebuild -r electron -t 22.0.0 -t 24.0.0 -t 25.0.0 --strip -u ${{ secrets.GH_TOKEN }}
```

or fix it like this

https://github.com/tree-sitter/node-tree-sitter/blob/d47af7824ab4cfd1c7195c4f9d8225104636a5db/binding.gyp#L73